examples/swap_networks.py

```diff
@@ -0,0 +1,83 @@
+"""Demonstrates swap networks."""
+
+import itertools
+import random
+
+import cirq
+import cirq.contrib.acquaintance as cca
+
+
+def get_random_graph(n_vertices, edge_prob=0.5):
```

Comment:
The code would be easier to read if the reader knew how vertices, edges etc are represented. Maybe add type annotations throughout.

This may have helped avoid the trap described by @babbush below.